Common subdirectories: xcoral-3.47/Doc and xcoral-3.47-r1/Doc
Common subdirectories: xcoral-3.47/Rpm and xcoral-3.47-r1/Rpm
Common subdirectories: xcoral-3.47/Smac and xcoral-3.47-r1/Smac
Common subdirectories: xcoral-3.47/SmacLib and xcoral-3.47-r1/SmacLib
diff -U5 -Bb xcoral-3.47/get_file.c xcoral-3.47-r1/get_file.c
--- xcoral-3.47/get_file.c	2000-04-12 15:37:22.000000000 -0400
+++ xcoral-3.47-r1/get_file.c	2015-02-18 10:51:30.009123690 -0500
@@ -1058,18 +1058,30 @@
 */
 void SetWindowName ( text )
     Text *text;
 {
     char *icon_name;
+    char hostname[64];
+    char windowtitle [MAXPATHLEN + 66];
     
-    icon_name = strrchr ( text -> filename, '/' );
+    gethostname(hostname, sizeof hostname);
+    hostname[63] = 0; /* make good and certain the hostname is numm-terminated */
+
+    snprintf(windowtitle,
+	     MAXPATHLEN + 66,
+	     "%s:%s",
+	     hostname,
+	     text -> filename);
+    
+    icon_name = strrchr ( windowtitle, '/' );
     if ( icon_name )
       icon_name ++;
     else
-      icon_name = text -> filename;
+      icon_name = windowtitle;
+    
     XSetIconName ( dpy,  text ->  w_parent, icon_name );
-    XStoreName ( dpy, text ->  w_parent, text -> filename );
+    XStoreName ( dpy, text ->  w_parent, windowtitle );
 }
 
 /*
 **	Function name : update_cwd
 **
diff -U5 -Bb xcoral-3.47/macros.c xcoral-3.47-r1/macros.c
--- xcoral-3.47/macros.c	1998-09-20 12:16:05.000000000 -0400
+++ xcoral-3.47-r1/macros.c	2015-02-18 10:51:30.010123656 -0500
@@ -48,10 +48,12 @@
 #  if HAVE_MALLOC_H
 #    include <malloc.h>
 #  endif
 #endif
 
+#include <strings.h> /* for bzero prototype */
+
 #include "main_text.h"
 #include "macros.h"
 #include "warn_box.h"
 #include "main_events.h"
 #include "get_file.h"
